smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/metric/fetcher/InotifyEventApplier.java

```diff
@@ -137,6 +137,7 @@ private String getCreateSql(Event.CreateEvent createEvent) throws IOException, M
         metaStore.insertFileDiff(fileDiff);
       }
     }
+    metaStore.deleteFileByPath(fileInfo.getPath());
```

Comment:
Delete & insert can be merge to [replace](https://github.com/Intel-bigdata/SSM/blob/trunk/smart-metastore/src/main/java/org/smartdata/metastore/dao/SmallFileDao.java#L57).